app/src/main/res/values/strings.xml

```diff
@@ -128,6 +128,11 @@
     <string name="app_components_checkbox_description_text">Checkboxes allows users to select one or multiple options from a list, toggle settings on or off, or confirm an action.</string>
     <string name="app_components_checkbox_error_label">Error</string>
 
+    <!-- Components: colored background -->
+    <string name="app_components_coloredBackground_label">Colored background</string>
+    <string name="app_components_coloredBackground_description_text">Colored backgrounds use surface colors to maximize the contrast with content.</string>
```

Comment:
Should we explain that components inside adapt their display according to the chosen background color?